Doc/library/typing.rst

```diff
@@ -578,6 +578,9 @@ These can be used as types in annotations and do not support ``[]``.
    * Every type is compatible with :data:`Any`.
    * :data:`Any` is compatible with every type.
 
+    .. versionchanged:: 3.11
+       :data:`Any` can now be used as a base class
```

Comment:
Can you add some explanation about the use cases?

This doesn’t say why someone could want that, nor does the ticket, one has to hunt for the last message in the mailing list thread!

Reply:
I added a sentence here. Let me know if I should make it longer; I tried to keep it short because this is a lot more niche than other things covered in typing.rst

Reply:
I think adding one short example might be nice

Reply:
Not sure about this. On the one hand an example would be useful; on the other hand it risks putting too much emphasis on a pretty obscure use case.

Reply:
Let's omit the example then (devguide says "err on the side of being succinct" for docs). I'm hoping to do some work on typing docs sometime soon, which might be a better home for such details.

Reply:
Alrighty!